martin/src/pg/utils.rs

```diff
@@ -13,6 +13,7 @@ use tokio::time::timeout;
 use crate::source::UrlQuery;
 
 #[cfg(test)]
+#[expect(clippy::ref_option)]
```

Comment:
CI is failing due to it not knowing this lint.

![image](https://github.com/user-attachments/assets/dda88965-e3eb-40f5-9c4b-2bebd64c1c01)

=> would it be an option to postpone this until CI is updated?

```suggestion
```

Reply:
Yep, totally not in a hurry:)

Reply:
Ps ironically, I'm the one who created that lint :)